Adicionado problemas 1160, 1164, 1165 e readme (#46)
petbccufscar/URI: Iniciante/1165/README.md

# Problema:   
Na matemática, um Número Primo é aquele que pode ser dividido somente por 1 (um) e por ele mesmo. Por exemplo, o número 7 é primo, pois pode ser dividido apenas pelo número 1 e pelo número 7.

**Problema Completo**: https://www.urionlinejudge.com.br/judge/pt/problems/view/1165

# Resolução:

> Iniciamos realizando a declaração das variaveis que iremos utilizar, dentre essas a `nCasos` irá armazenar a quantidade de casos de teste que iremos realizar, este valor será passado no começo, portanto logo em seguida utilizamos a função `scanf` para receber o valor. Duas variaveis `i` e `j` que serão utilizadas como contadores durante os laços de repetição `for` que iremos utilizar. Teremos ainda uma variavel `valor` que a cada iteração irá receber um valor para ser verificado a condição de primo, para isso utilizaremos uma variavel `primo` que utilizaremos esta usando os valores `0 ou 1` indicando ser ou não primo.
> 

Reviewer: ```suggestion
Iniciamos realizando a declaração das variáveis que iremos utilizar, dentre essas a `nCasos` irá armazenar a quantidade de casos de teste que iremos realizar, este valor será recebido no começo, portanto logo após a declaração de variáveis, utilizamos a função `scanf` para recebe-lo. Duas variáveis `i` e `j` que serão utilizadas como contadores durante os laços de repetição `for` que iremos utilizar. Teremos ainda uma variável `valor` que a cada iteração irá receber um valor para ser verificado a condição de primo, para isso utilizaremos a variável `primo` que assumirá valores `0 ou 1` indicando ser primo (1) ou não primo (0).

```

```c
int nCasos, valor, i, j, primo;
scanf("%d", &nCasos);
```

> Em nosso laço de repetição principal realizamos a leitura do valor que será passado e definimos também inicialmente este como primo, pois nas etapas futuras caso este não seja primo esse valor é alterado

Reviewer: ```suggestion
Em nosso laço de repetição principal realizamos a leitura do `valor` que será passado e definimos também inicialmente este como primo, pois nas etapas futuras teremos verificações que invalidam a condição de primo, alterando o valor da variável `primo` para 0.
```

```c

> for(i=0; i<nCasos; i++){
> 	scanf("%d", &valor);
> 	primo = 1;

Reviewer: ```suggestion
for(i=0; i<nCasos; i++){
	scanf("%d", &valor);
	primo = 1;
	if(valor == 1)
            primo = 0;
	
```

```

> 

Reviewer: ```suggestion
Observe que verificamos aqui se o `valor` recebido é 1 e caso positivo, setamos a variável primo para 0, já que o número 1 não é primo, pois apesar de ser divisível por 1 e ele mesmo, não possui 2 divisores.  
```

Após recebido o valor utilizamos a variavel `j` como contador indo de 2 até o valor, verificando a condição de não possuir divisor, o que representa a condição de ser um número primo. Caso encontre um divisor, definimos a variavel `primo` como 0
```c
for(j=2; j<valor; j++)
	if(valor % j == 0)
		primo = 0;
```

> Ao final realizamos a verificação do valor final contido na variavel `primo`, caso este seja igual a 1, imprimimos que o número é primo, caso contrário imprimimos que o valor não é primo.

Reviewer: ```suggestion
Ao final realizamos a verificação do valor final contido na variável `primo`, caso este seja igual a 1, significa que após dividirmos este valor por todos os números que o antecedem, partindo de 2, ele não foi divisível por nenhum, portanto, só é divisível por 1 e ele mesmo (condição de existência de um número primo). Assim, imprimimos que o número é primo e caso ele encontre ao menos um número divisor além de 1 e ele mesmo, imprimimos que o valor não é primo.
```
